MongoPocWebApplication1/ControllersPresentationAndApplication/CountryController.cs

```diff
@@ -28,5 +29,31 @@ public async Task<ActionResult<string>> Post(Country country)
             await countryRepository.AddAsync(country);
             return country.Id;
         }
+
+        [HttpPatch]
+        public async Task<IActionResult> PatchCountryWithModelState(
+            [FromBody] JsonPatchDocument<Country> patchDoc, string id)
+        {
+            if (patchDoc != null)
```

Comment:
Implement json merge patch.

BTW, we know that in 90+ cases FE should send only one field as they use implicit save.
That allows us to simplify the task and do the next:

We expose an endpoint PUT|PATCH service/{id}/fields/{fieldName} that receives a payload {"value": number|string|array...}
You can start from that instead of a much more complicated JSON merge patch scenario.

Reply:
I agree, this should be very simple to implement and this is how I did it in the past, I just wanted to check a known convention pattern  for patch. we could use the DTO instead the Entity on the API.
But this could be added later on for cases of more complex update if needed.